ceph/ceph_admin/restart.py

```diff
@@ -29,8 +29,7 @@ def restart(self: Union[OrchProtocol, DaemonProtocol], config: Dict):
         restart_init_time = datetime.datetime.utcnow()
         rc = self.op("restart", config)
         args = config.get("args")
```

Comment:
not important... we could do `args = config.get("args", {})`.

This would avoid `if args and args.get("verify", True)`

`verify` being an internal usage... it was removed from the args dictionary. This allows to process `args` directly 

IMO, we should modify this line and leave the rest.